InternationalSpaceStationTracker.Tests/Services/SatelliteServiceTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using InternationalSpaceStationTracker.Services;
+using InternationalSpaceStationTracker.Tests.Data;
+using Moq;
+using Moq.Protected;
+using NUnit.Framework;
+using System.Net;
+
+namespace InternationalSpaceStationTracker.Tests.Services
+{
+    public class SatelliteServiceTests
+    {
+        private SatelliteService _satelliteService;
+        private HttpClient _httpClient;
+        private Mock<HttpMessageHandler> _handlerMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            _handlerMock = new Mock<HttpMessageHandler>();
+            _httpClient = new HttpClient(_handlerMock.Object);
+            _satelliteService = new SatelliteService(_httpClient);
+        }
+
+        [Test]
+        public async Task GetSatellites_ReturnsDataWithExpectedProperties()
+        {
+            _handlerMock.Protected()
+                .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
+                .ReturnsAsync(new HttpResponseMessage()
```

Comment:
Use the constructor argument rather than the object initialiser where possible. Inside it's getting a default value which then gets overridden.

Reply:
thanks